resources/carrier/zh/86.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2016 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Note 86170 is used by virtual operators, and their individual assignments are
```

Comment:
Same here: need a comment about where the translations come from, rather than this note about 86170 which is more relevant in the source file (in this case en)